class-15.md

```diff
@@ -0,0 +1,31 @@
+# Intro to Blockchain Development
+
+### Class 15 - Exchanges 
+
+Many users don't always hold cryptocurrency inside a wallet or linked to a single address. In many instances, they purchase on an exchange, or in a system where one party takes custody of cryptocurrency for many users. 
+
+Reading & Videos:
+
+- [Hot vs. Cold Wallets](https://youtu.be/Aji_E9sw0AE)* - An Andreas Antonopoulos talk about hot and cold wallets
+
+### Intro To Exchanges
+
+While it is possible to get cryptocurrency by through the transaction fee or mining reward in a block, the majority of Bitcoin and Ethereum users do not perform mining themselves. Additionally, many miners make payments to electricity companies using fiat currencies like the renminbi or sometimes the dollar. Many miners need to sell holdings of cryptocurrency wish to sell their holdings, while many end users who wish to transact or invest purchase it. This results in a marketplace where users need to transact between traditional fiat currencies and cryptocurrencies.
```

Comment:
`While it is possible to get cryptocurrency by through the transaction fee or mining reward in a block`
->
While it is possible to get cryptocurrency by transaction fees or mining rewards in a block

Reply:
`the majority of Bitcoin and Ethereum users do not perform mining themselves`
-> the majority of Bitcoin and Ethereum holders do not mine the coins themselves

Reply:
`Many miners need to sell holdings of cryptocurrency wish to sell their holdings, while many end users who wish to transact or invest purchase it.`

-> This doesn't make sense :(